utilities/ideascale-importer/README.md

```diff
@@ -0,0 +1,45 @@
+IdeaScale Importer
+===
+
+## Getting Started
+
+> *Python 3 is required*
+
+To install all the dependencies without polluting your Python installation run:
+
+```sh
+pip install -r requirements.txt
+```
+
```

Comment:
Actually you would need to be inside a pyenv to no pollute your local  python:


```suggestion

First: Install [pyenv](https://github.com/pyenv/pyenv#installation) AND [pyenv virtualenv](https://github.com/pyenv/pyenv-virtualenv).

Then: Create a virtual environment and install dependencies:

```sh
pyenv install 3.11.1
pyenv virtualenv 3.11.1 ideascale-importer-venv-3.11.1
pyenv  activate ideascale-importer-venv-3.11.1
pip install -r requirements.txt
```

```